apps/portal/components/pagesComponents/_imagesScreen/components/CreateFolderModal.tsx

```
@@ -164,14 +168,14 @@ const CreateFolderModal: React.FC<ICreateFolderModalProps> = ({
         maxLength={35}
       />
       <div className="tw-mt-4 tw-text-lg tw-ml-1">
-        Select folder{' '}
+        {t('modals.selectFolder')}{' '}
         <span className="tw-font-semibold" style={{ color: color }}>
-          color
+          {t('modals.color')}
         </span>
       </div>
       <div className="tw-flex tw-items-center tw-space-x-4"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)